frontend/src/bundles/auth/components/sign-up-form/sign-up-form.tsx

```diff
@@ -62,34 +62,48 @@ const SignUpForm: React.FC<Properties> = ({ onSubmit }) => {
                 className={styles.registration__form}
                 onSubmit={handleFormSubmit}
             >
-                <FormGroup error={errors.firstName} label="First Name">
+                <FormGroup
+                    className={styles.form_element_1}
```

Comment:
Consider changing to a more meaningful class name

Reply:
Suggest something please :-)
grid_element_1, grid_element_2 etc. will be fine?